src/popup/snooze.scss

```diff
@@ -1,34 +1,82 @@
 @import 'node_modules/rc-calendar/assets/index';
 @import 'node_modules/rc-time-picker/assets/index';
 
-$panel-width: 250px;
+$panel-width: 320px;
+$panel-height: 474px;
+$panel-shadow-width: 13px;
+
+$narrow-panel-width: 230px;
+$narrow-panel-height: 550px;
```

Comment:
I think this depends on how much stuff you have in your menu panel…  (Try adding a ton of buttons to it, and see what happens.)